src/detours.cpp

```diff
@@ -1524,6 +1524,24 @@ struct DetourThread
 {
     DetourThread *      pNext;
     HANDLE              hThread;
+    BOOL                fCloseThreadHandleOnDestroy;
```

Comment:
If we're giving this class ownership semantics, delete the copy constructor and assignment operator.